server/README.md

```diff
@@ -135,6 +135,18 @@ For the most up-to-date view of the database, see the db/schema.db file.
 
 [Update the database schema overview chart](https://www.lucidchart.com/documents/edit/164bd2c8-40ed-4797-acd6-e7d0e0a9c0d1/0_0?shared=true) using this link.
 
+Some useful commands for handling databases. 
+
+Roll back one db migration:
+```
+rake db:rollback STEP=1
```

Comment:
It's possible and valid to use the rake tasks via rake. It's also possible to use the rails command, instead, which would follow Rails' documentation

Reply:
(STEP defaults to 1.)